packages/pluggableWidgets/document-viewer-web/src/components/DocumentViewer.tsx

```diff
@@ -10,8 +10,17 @@ export interface DocumentViewerProps {
 
 export function DocumentViewer({ className, src, rootStyle, tabIndex }: DocumentViewerProps): ReactElement {
     return (
-        <div className={classnames("widget-document-viewer", className)} style={rootStyle} tabIndex={tabIndex}>
-            <object className="widget-document-viewer-content" data={src} type="application/pdf">
+        <div
+            className={classnames("widget-document-viewer", className)}
+            style={{ ...rootStyle, position: "relative" }}
+            tabIndex={tabIndex}
+        >
+            <object
+                className="widget-document-viewer-content"
+                data={src}
+                type="application/pdf"
+                style={{ position: "absolute" }}
```

Comment:
CSP doesnt like it inline styles 🙊

Reply:
@r0b1n can you check whether this is relevant? To do so, you can either 1) run this in a strict CSP enabled project and see if it throws errors or 2) compile the widget and see if this styling is embedded inside the widget.

Reply:
This way of working with styles in `react` is not in conflict with any CSP directives as `react` internally uses APIs for inline styling that is considered safe.